style.css

```diff
@@ -1,71 +1,92 @@
 body {
-    background-color: antiquewhite;
-    padding: 40px, 0;
     font-family: Arial, Helvetica, sans-serif;
+    background-color: whitesmoke;
+    padding: 40px 0;
+}
 
+header {
+    display: flex;
 }
 
-span,
+h1,
 p,
-h4 {
-    color: grey;
+h4,
+h5 {
+    color: gray;
 }
 
-.container {
-    width: 600px;
-    height: 400px;
-    margin: 200 auto;
-    background-color: white;
+h1,
+h2,
+h3,
+h4,
+h5,
+p {
+    margin-left: 30px;
 }
 
-img {
-    width: 300px;
-    height: 400px;
+p {
+    line-height: 1.5;
+    padding-right: 128px;
+    margin-top: 10px;
+    margin-bottom: 10px;
 }
 
-.perfume-header {
-    display: flex;
-    align-items: center;
-}
+h1 {
 
+    margin-top: 20px;
+    font-size: 15px;
+    margin-bottom: 10px;
+}
 
 h2 {
-    font-size: 40px;
-    margin-left: 30px;
-    color: green;
+    margin-bottom: 10px;
+    margin-top: 20px;
+    font-size: 35px;
+    padding-right: 100px;
 }
 
 h3 {
-    border: 2px solid green;
-    color: white;
-    text-decoration: none;
-    padding: 10px 70px;
-    margin-left: 30px;
-    display: inline-block;
-    border-radius: 10px;
-    background-color: green;
+    font-size: 40px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    color: green;
 }
 
-h3:focus {
-    outline: none;
-    color: white;
-    background-color: green;
-}
 
-h3:active {
-    text-decoration: underline;
+.container {
+    width: 700px;
+    margin: 0 auto;
+    background-color: white;
+    border-radius: 20px;
 }
 
-h4 {
-    margin-left: 25px;
+.image {
+    width: 330px;
+    height: 450px;
+    border-top-left-radius: 20px;
+    border-bottom-left-radius: 20px;
+}
 
+.shopping-icon {
+    background-color: green;
+    width: 25px;
+    height: 25px;
 }
 
 .price {
     display: flex;
-    align-items: center;
 }
 
-.discription {
-    margin-left: 20px;
+.link {
+    border: 4px solid green;
+    color: white;
+    background-color: green;
+    border-radius: 10px;
+    display: inline-block;
+    margin-left: 30px;
+    padding: 5 50 5 50;
+    font-size: 21px;
+    margin-top: 10px;
+    margin-bottom: 10px;
```

Comment:
instead of 4 margin rules you can use a shortcut

`margin: topValue, bottomValue, rightValue, leftValue;`